seed/views/v3/import_files.py

```diff
@@ -157,15 +157,15 @@ def retrieve(self, request, pk=None):
         })
 
     @swagger_auto_schema_org_query_param
-    @has_perm_class('requires_viewer')
     @api_endpoint_class
     @ajax_request_class
+    @has_perm_class('requires_viewer')
```

Comment:
has_perm_class must come "after" the other decorators, ie it must be wrapped itself by the other decorators. This is necessary b/c the api_endpoint_class decorator needs to configure the request.user before we check authz